marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -95,6 +95,11 @@ class BrowsePage {
    * available facets are 'collection', 'created-on', 'job-id', 'flow', 'step'
    */
 
+  
+  getSideBarContainer() {
+    return cy.get('#sideBarContainer');
```

Comment:
@timur-isangulov Please remove these methods too which are not being used

Reply:
Removed.